src/paperbot/api/routes/runbook.py

```
@@ -1,48 +1,52 @@
-from paperbot.infrastructure.monitoring.resource_monitor import get_resource_monitor
-from paperbot.infrastructure.stores.models import AgentRunModel, ArtifactModel, Base, RunbookStepModel
+from paperbot.infrastructure.stores.models import AgentRunModel, ArtifactModel, Base
 from paperbot.infrastructure.stores.sqlalchemy_db import SessionProvider
 from paperbot.repro.e2b_executor import E2BExecutor
 from paperbot.repro.execution_result import ExecutionResult
+from paperbot.repro.openai_ci_executor import OpenAICodeInterpreterExecutor
 
 router = APIRouter()
 
-ExecutorType = Literal["docker", "e2b"]
+ExecutorType = Literal["docker", "e2b", "openai_ci"]
```

Ruff (0.15.1)

[error] Undefined name `Literal`

(F821)